product_variants_no_automatic_creation/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Prevent the automatic creation of the variants of a product
```

Comment:
The infinitive can be used in case of an imperative. The title can also be interpreted that way maybe...